src/Cli/dotnet/Commands/Run/VirtualProjectBuildingCommand.cs

```diff
@@ -88,8 +84,8 @@ public VirtualProjectBuildingCommand(
     public override int Execute()
     {
         Debug.Assert(!(NoRestore && NoBuild));
-        var consoleLogger = RunCommand.MakeTerminalLogger(Verbosity);
-        var binaryLogger = GetBinaryLogger(BinaryLoggerArgs);
+        var consoleLogger = TerminalLogger.CreateTerminalOrConsoleLogger(LoggerArgs);
```

Comment:
Did `RunCommand.MakeTerminalLogger` not create a terminal logger, you ask? No:

https://github.com/dotnet/sdk/blob/a37b3513eeba8fa1377f42d9db87c5dca1ae6821/src/Cli/dotnet/Commands/Run/RunCommand.cs#L397-L404

But I'm not confident that should change, because that's used for evaluating the project after it's built and if terminal logger is used there, it results in additional "Build succeeded in 0.0s" message like:

![image](https://github.com/user-attachments/assets/1f5495b4-8d04-43ca-a097-b8132bd0f632)

whereas if the evaluation phase continues to use the console logger, it's nicely quiet.